18.05.2015 03:30:59 - INFO: Alarm!
18.05.2015 03:30:59 - DEBUG: Loading plugin template
18.05.2015 03:30:59 - DEBUG: Throw Template Plugin
18.05.2015 03:30:59 - INFO: ZVEI: 12345 wurde auf 80000000 empfangen!
18.05.2015 03:30:59 - DEBUG: try 5/0
18.05.2015 03:30:59 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 8, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 03:31:00 - INFO: Alarm!
18.05.2015 03:31:00 - DEBUG: Loading plugin template
18.05.2015 03:31:00 - DEBUG: Throw Template Plugin
18.05.2015 03:31:00 - INFO: ZVEI: 12345 wurde auf 80000000 empfangen!
18.05.2015 03:31:00 - DEBUG: try 5/0
18.05.2015 03:31:00 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 8, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 03:31:01 - INFO: Alarm!
18.05.2015 03:31:01 - DEBUG: Loading plugin template
18.05.2015 03:31:01 - DEBUG: Throw Template Plugin
18.05.2015 03:31:01 - INFO: ZVEI: 12345 wurde auf 80000000 empfangen!
18.05.2015 03:31:01 - DEBUG: try 5/0
18.05.2015 03:31:01 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 8, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 03:31:02 - INFO: Alarm!
18.05.2015 03:31:02 - DEBUG: Loading plugin template
18.05.2015 03:31:02 - DEBUG: Throw Template Plugin
18.05.2015 03:31:02 - INFO: ZVEI: 12345 wurde auf 80000000 empfangen!
18.05.2015 03:31:02 - DEBUG: try 5/0
18.05.2015 03:31:02 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 8, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 03:31:03 - INFO: Alarm!
18.05.2015 03:31:03 - DEBUG: Loading plugin template
18.05.2015 03:31:03 - DEBUG: Throw Template Plugin
18.05.2015 03:31:03 - INFO: ZVEI: 12345 wurde auf 80000000 empfangen!
18.05.2015 03:31:03 - DEBUG: try 5/0
18.05.2015 03:31:03 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 8, in run
    test = 5/0
ZeroDivisionError: division by zero
